### DISCUSSION TITLE
How to generate standalone Gradio application with the trained model ?

### DISCUSSION THREAD

**Discussion by billyjoker** [TheLastBen/fast-stable-diffusion#1569] [General]
Hi I do not know if this is the place to ask this question. I did some trained models successfully but I see that there is a time limit that when it exceeded, the app get disconnected automatically. I heared hat it is possible to build the gradio application as standalone, so you can share that specific software with others. Where is the guide to perform this one or some steps I should to do? 
Thanks in advance

**Comment by TheLastBen** [comment]
You need a GPU to run the model and google colab provides that, unless you have a GPU, then you can install A1111 locally and run the model.

**Reply by billyjoker** [reply]
Ok thanks @TheLastBen, I'd like to try it!  Is there a guide to install A1111 locally ?

**Reply by TheLastBen** [reply]
The instructions are on the main page of the A1111 repo https://github.com/AUTOMATIC1111/stable-diffusion-webui
